.gitignore

```diff
@@ -2,4 +2,10 @@ lib
 obj
 bin
 .DS_Store
-.#*#
\ No newline at end of file
+<<<<<<< HEAD
+.#*#
+a_dot
+moving_triangle
+=======
+.#*#
+>>>>>>> flyx/master
```

Comment:
This is a conflict you failed to solve after merging master into your branch. Only lines 6-8 should stay.